tests/src/test/scala/org/apache/openwhisk/core/cli/test/WskCliBasicUsageTests.scala

```diff
@@ -1183,6 +1188,60 @@ class WskCliBasicUsageTests extends TestHelpers with WskTestHelpers {
       .stderr should include("Cannot save action sequences")
   }
 
+  it should "create an action, and get its individual fields" in withAssetCleaner(wskprops) {
```

Comment:
This test was moved here so that it can access the protected `Parameters` functionality.